shpc/main/registry/remote.py

```diff
@@ -190,6 +190,8 @@ def _update_cache(self, force=False):
         if self._cache and not force:
             return
 
+        if self.source.startswith("ssh"):
```

Comment:
We could do the same here, e.g., `if re.search("<pattern for acceptable protocols">, self.source):`